docs/foundry/getting-started.md

```diff
@@ -0,0 +1,317 @@
+# Getting Started
+
+In this guide, you'll learn how to deploy, upgrade, and test an upgradeable contract using ChugSplash.
+
+## Table of Contents
+
+1. [Setup a Foundry project](#1-setup-a-foundry-project)
+2. [Install ChugSplash](#2-install-chugsplash)
+3. [Configure your `foundry.toml` file](#3-configure-your-foundrytoml-file)
+4. [Update remappings](#4-update-remappings)
+5. [Create a contract](#5-create-a-contract)
+6. [Create a ChugSplash file](#6-create-a-chugsplash-file)
+7. [Create your deployment script](#7-create-your-deployment-script)
+8. [Deploy with ChugSplash](#8-deploy-with-chugsplash)
+9. [Test with ChugSplash](#9-test-with-chugsplash)
+10. [Upgrade with ChugSplash](#10-upgrade-with-chugsplash)
+
+## Prerequisites
+
+The following must be installed on your machine:
+- [Node.js >=v15 and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+- [Foundry](https://book.getfoundry.sh/getting-started/installation)
+
+You must also have a basic understanding of how to use Foundry. [See here](https://book.getfoundry.sh/getting-started/first-steps) for a brief introduction.
+
+## 1. Setup a Foundry project
+
+If you have an existing foundry project, navigate to it then [skip to step #2](#2-install-chugsplash).
+
+If you're starting a new project, run:
+
+```
+forge init hello_foundry && cd hello_foundry
+```
+
+Then, delete the files that come with the default Foundry project:
+```
+rm src/Counter.sol script/Counter.s.sol test/Counter.t.sol
+```
+
+## 2. Install ChugSplash
+
+In your project root, run:
+
+```
+npm install @chugsplash/plugins
+```
+
+or
+
+```
+yarn add @chugsplash/plugins
+```
+
+You may also want to add `node_modules` to your .gitignore file.
+
+## 3. Configure your `foundry.toml` file
+
+Edit your `foundry.toml` file to include all of the following options. If you leave any of these out, ChugSplash will not work properly.
+
+```
+[profile.default]
+out = 'out'
+ffi = true
+build_info = true
+extra_output = ['storageLayout']
+force = true
+fs_permissions = [{ access = "read", path = "./"}]
+
+remappings = [
+  'forge-std/=node_modules/forge-std/src/',
+  'ds-test/=node_modules/ds-test/src/',
+  'chugsplash/=node_modules/@chugsplash/plugins/dist/contracts/'
+]
```

Comment:
It's redundant to have both a `remappings` field in `foundry.toml` as well as a `remappings.txt` file, so we can just remove this parameter here